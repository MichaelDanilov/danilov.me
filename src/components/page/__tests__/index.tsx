import React from 'react';
import renderer from 'react-test-renderer';
import Page from '..';

describe('Page', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Page>
          <span>Lorem ipsum</span>
        </Page>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
