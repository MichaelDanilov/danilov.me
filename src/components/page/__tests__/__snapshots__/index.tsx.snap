// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders correctly 1`] = `
<div
  className="sc-AxjAm bMqUCa"
>
  <span>
    Lorem ipsum
  </span>
</div>
`;
